.travis.yml

```diff
@@ -13,6 +13,7 @@ cache:
   pip:
   directories:
     - dynamodb
+    - minio
```

Comment:
fair enough, but Minio is not real s3 and just one more thing to understand going forwards